plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -65,6 +76,8 @@ def __init__(self, repo=None, file_path=None, config=None, metadata=None, type_i
             self._handle_image()
         elif type_id == models.Tag._content_type_id.default:
             self._handle_tag(metadata)
+        elif type_id == models.Blob._content_type_id.default:
```

Comment:
i still think that the type should be manifest, and therefore the method _handle_manifest, or even better _handle_image_manifest, you never know maybe at some point we would add support for manifest list upload.